doc/developer_docs.md

```diff
@@ -13,3 +13,5 @@ Contents:
  - [Development tips](development_tips.md#development-tips)
  - [Option normalizers](option_normalizers.md#option-normalizers)
  - [Internationalization](i18n.md#internationalization)
+
+Slides from tutorial at Config Management Camp 2015: [Hacking with hammer](https://tstrachota.fedorapeople.org/slides/hacking_with_hammer/)
```

Comment:
Great slides @tstrachota!!!  :clap: 
I'd separate this into section "Links". I took me a while to figure out how to navigate through the slides. It would be worth it to add some hint on the first page.